### ISSUE TITLE
Remap error for java 21

### ISSUE THREAD

**Issue by AaronR92** [Alvinn8/paper-nms-maven-plugin#22]
When i try to compile my plugin with java 21
## pom.xml
```xml
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.11.0</version>
                <configuration>
                    <source>21</source>
                    <target>21</target>
                </configuration>
            </plugin>
```

It says:
```cmd
[ERROR] Failed to execute goal ca.bkaw:paper-nms-maven-plugin:1.4.2:remap (default) on project auroramainplugin: Execution default of goal ca.bkaw:paper-nms-maven-plugin:1.4.2:remap failed: java.lang.RuntimeException: error analyzing A:\Projects\Java\Aurora-Main-Plugin\target\classes\io\github\aaronr92\auroraproject\AuroraPlugin.class from A:\Projects\Java\Aurora-Main-Plugin\target\classes: Unsupported class file major version 65 -> [Help 1]
```
## Error screenshot
![image](https://github.com/Alvinn8/paper-nms-maven-plugin/assets/86726028/e5051dd0-fb7d-4aac-9b2b-fffd699f4b82)

**Comment by Alvinn8** [comment]
Hello, I’m on my phone right now. Could you try the latest snapshot version of the plugin? I updated some dependencies a while ago so that should support the latest java version. I can push that as a release when I come home.

**Comment by AaronR92** [comment]
sure, i can

**Comment by Alvinn8** [comment]
Sorry, it seems I never pushed those changes I was referring to. Try version `1.4.3-SNAPSHOT` now and it should work. I'll push that as version `1.4.3` soon.

**Comment by AaronR92** [comment]
Yeah, it works! Just asked to try it out and now it works fine.

### Thank you 😊

**Comment by Alvinn8** [comment]
Great, thanks! I've released the changes as version `1.4.3` now.
